.gitignore

```diff
@@ -1,3 +1,4 @@
 node_modules/
 package-lock.json
 var/
+temp/
```

Comment:
Should this have been checked in?

Reply:
yes, temp is generated by the script. It is cleaned up - but can get left behind if there is a local issue.

Reply:
no longer valid since theme is now a dependency